frontend/app/components/themeProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react"
+import theme from "@/styles/theme"
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles"
```

Comment:
Why are you renaming `ThemeProvider` to `MuiThemeProvider`?

Reply:
we renamed it because we the code gets complex and we start using differnt ui ThemeProvider will be a common name , so to avoid this conflict we had named ThemeProvider to MuiThemeProvider